addon/components/sl-date-time.js

```diff
@@ -169,8 +185,14 @@ export default Ember.Component.extend( TooltipEnabled, {
         'timezone',
         'momentValue',
         function() {
-            return this.get( 'momentValue' )
-                .tz( this.get( 'timezone' ) ).format( 'z' );
+            const momentValue = this.get( 'momentValue' );
+            const timezone = this.get( 'timezone' );
+
+            if ( momentValue && timezone ) {
+                return momentValue.tz( timezone ).format( 'z' );
+            }
+
+            return null;
```

Comment:
I have created https://github.com/softlayer/sl-ember-components/issues/701 to address a deficiency with this component.  With this issue in place `this.get( 'timezone' )` will also return a correct value meaning that this computed property will also always return a value, negating the need to modify the code to return `'null`.  Your changes can be removed.